project.mml

```diff
@@ -1524,7 +1524,7 @@ Layer:
                                           OR (boundary = 'protected_area' AND tags->'protect_class' IN ('1','1a','1b','2','3','4','5','6'))
                                           THEN boundary END,
                 'leisure_' || CASE WHEN leisure IN ('nature_reserve') THEN leisure END,
-                'tourism_' || CASE WHEN tourism IN ('information') THEN tourism END,
+                'tourism_' || CASE WHEN tourism IN ('information') AND tags->'information' IN ('audioguide', 'board', 'guidepost', 'office', 'map', 'terminal') THEN tourism END,
```

Comment:
tactile_map is missing in the list of values here.

Reply:
Thanks, corrected the mistake and tested